core/cli/src/informant.rs

```diff
@@ -47,7 +47,7 @@ pub fn start<C>(service: &Service<C>, exit: ::exit_future::Exit, handle: TaskExe
 	let display_notifications = network.status().for_each(move |sync_status| {
 
 		if let Ok(info) = client.info() {
-			let best_number: u64 = info.chain.best_number.as_();
+			let best_number = info.chain.best_number.saturated_into::<u64>();
```

Comment:
Shouldn't it here be sufficient to add a bound for `Display + Serialize`?

Reply:
yes, but don't want to do it in this PR.